src/ui/components/ProposalCallInfo.tsx

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
 import {StyleSheet, View} from 'react-native';
-import {Paragraph, Card, Caption} from '@ui/library';
+import {Paragraph, Card, Caption, Text} from '@ui/library';
 import {Padder} from '@ui/components/Padder';
 import type {DemocracyProposal, DemocracyReferendum, ProposalSubCall} from 'src/api/hooks/useDemocracy';
 import {standardPadding} from '@ui/styles';
+import {MotionProposal} from 'src/api/hooks/useCouncilMotions';
 
 type ProposalInfoProps = {
-  proposal: DemocracyProposal | DemocracyReferendum;
+  proposal: DemocracyProposal | DemocracyReferendum | MotionProposal;
```

Comment:
This component should also display some more information such as `proposer`, `beneficiary`, `payout` etc.

But before that we need to refactor the graph a little bit. Right now the motion proposal resolver is using the `AccountsService` in the backend to get the proposer account data and the democracy proposal resolver is using resolver chaining to get the account information data for proposer. That's why the response shape is different.

I am creating the necessary tickets and can be taken care of as a follow-up after merging this.